src/test/java/org/broadinstitute/hellbender/engine/ReferenceMultisourceUnitTest.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.hellbender.engine;
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import org.apache.spark.SparkConf;
+import org.apache.spark.serializer.KryoSerializer;
+import org.apache.spark.serializer.SerializerInstance;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceWindowFunctions;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import scala.reflect.ClassTag;
+import scala.reflect.ClassTag$;
+
+public class ReferenceMultisourceUnitTest extends BaseTest {
+
+    private static String twoBitRefURL = publicTestDir + "large/human_g1k_v37.20.21.2bit";
```

Comment:
I see now that it probably is specific to `ReferenceMultiSource` -- comment withdrawn.